sources/add_config.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# $1 is APP
+# $2 is OIDC_NAME
+# $3 is OIDC_CALLBACK
+# $4 is OIDC_SECRET
```

Comment:
Rather than putting comments, what do you think of creating variables and even use [parameter expansions](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) to control how they are used and give them default values?

```suggestion
APP=$1
OIDC_NAME=$2
OIDC_CALLBACK=${3/#https:\/\//} # Remove the scheme of the URL if specified, as it will be set anyways again below.
OIDC_SECRET=${4:-$(ynh_string_random --length=32 --filter='A-F0-9')} # If the secret is not passed, generate one
```